src/main/java/de/tum/in/www1/artemis/service/dto/athena/ModelingFeedbackDTO.java

```diff
@@ -0,0 +1,32 @@
+package de.tum.in.www1.artemis.service.dto.athena;
+
+import java.util.List;
+
+import jakarta.validation.constraints.NotNull;
+
+import de.tum.in.www1.artemis.domain.Feedback;
+
+/**
+ * A DTO representing a Feedback on a ModelingExercise, for transferring data to Athena and receiving suggestions from Athena
+ */
+public record ModelingFeedbackDTO(long id, long exerciseId, long submissionId, String title, String description, double credits, Long structuredGradingInstructionId,
```

Comment:
Is there anything specific about modelling exercises? Why can't we use a single DTO for all exercise types?

Reply:
The ` List<String> elementIds` attribute as well as the implementation of the `of` method are slightly different - but the difference is admittedly quite minimal 🤔